index.html

```diff
@@ -8,12 +8,117 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="karma-store" href="/store.html">
 </head>
+
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    <header>
+        <img class="svg-logo"
+            src="./img/karma-logo.svg" 
+            alt="Karma company logo">
+        </img>
+        <nav class="nav-bar">
+            <ul class="navigation__list">
+                <li class="active">Meet Karma</li> 
+                <li>How it Works </li>
+                <li><a href="./store.html">Store</a></li>
+                <li>Blog</li>
+                <li>Help</li>
+                <li>Login</li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+                
+            </div>
+        </nav>              
+    </header>
+
+    <!-- <main> -->
+    
+        <section class="hero-intro">
+            <div class="section-content">
+                <h1>Introducing Karma</h1>
+                <h3>Bring WiFi with you, everywhere you go.</h3>
+                <div class="btn">
+                    <button>Learn More</button>
+                </div>   
+            </div>
+
+        </section>
+
+        <section class="main-icons">
+            <h2>Everyone needs a little Karma</h2>
+            <div class="container-icon">
+                <div class="karma-icon">
+                    <img class="karma-icon-svg" 
+                        src="img/icon-devices.svg"
+                        alt="devices icon">
+                    </img>
+                    <h2>Internet for all devices</h2>
+                </div>
+                <div class="karma-icon">
+                    <img class="karma-icon-svg" 
+                        src="img/icon-coffee.svg"
+                        alt="fresh coffee icon">
+                    </img>
+                    <h2>Boost your productivity</h2>
+                </div>
+                <div class="karma-icon">
+                    <img class="karma-icon-svg" 
+                        src="img/icon-refill.svg"
+                        alt="refill icon">
+                    </img>
+                    <h2>Pay as You Go</h2>
+                </div>            
+            </div>        
+        </section>
 
+    <!-- additional -->
+        <section class="additonal-features">
+            <div class="karma-feature-img">
+                <img class="feature-img"
+                src="additional/feature-img.png"
+                alt="Smiling lady using karma on her tablet">
+            </div>
+            <div class="karma-content">
+                <h2>Wherever I am, I just don't worry about my connection anymore!</h2>
```

Comment:
you could use the `blockquote` element here for better semantics https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote